### PR TITLE
feat: upgrade fundamental-styles to 0.8.0-rc.16

### PR DESCRIPTION
**Pull request by meganaconley**
### Description
* Update Calendar to match spec and add screen-reader-only section
* Style changes to RTL checkbox/radio

Calendar before:

Screen Shot 2020-04-02 at 5 02 07 PM

Calendar after:

Screen Shot 2020-04-02 at 4 58 43 PM

Screen Shot 2020-04-02 at 5 02 07 PM

Radio/checkbox before:

Screen Shot 2020-04-02 at 5 08 16 PM

Screen Shot 2020-04-02 at 5 08 10 PM

Radio/checkbox after:
Screen Shot 2020-04-02 at 5 06 31 PM

Screen Shot 2020-04-02 at 5 06 23 PM

### CODE REVIEW

File: src/Calendar/Calendar.js [hunk 1]
```diff
@@ -255,7 +260,8 @@ class Calendar extends Component {
                 return (
                     <td aria-selected={isSelected} className={calendarItemClasses}
                         key={month} name={month}
-                        onClick={() => this.changeMonth(month)}>
+                        onClick={() => this.changeMonth(month)}
+                        onFocus={this.handleMonthFocus(month)}>
```
Comment (skvale) [inline src/Calendar/Calendar.js:264]: should this be `onFocus={() => this.handleMonthFocus(month)}>`?

File: src/Calendar/Calendar.js [hunk 2]
```diff
@@ -309,7 +317,8 @@ class Calendar extends Component {
                     <td aria-selected={isSelected}
                         className={yearClasses} key={element}
                         name={element}
-                        onClick={() => this.changeYear(element)}>
+                        onClick={() => this.changeYear(element)}
+                        onFocus={this.handleYearFocus(element)}>
```
Comment (skvale) [inline src/Calendar/Calendar.js:321]: should this be `onFocus={() => this. handleYearFocus(element)}>`?

File: src/Calendar/Calendar.js [hunk 3]
```diff
@@ -359,6 +368,19 @@ class Calendar extends Component {
         }
     }
 
+    handleDayFocus = date => () => {
+        this.setState({ currentFocusDay: date });
+    }
+
+    handleMonthFocus = month => () => {
+        console.log(month) /* eslint-disable-line */
```
Comment (skvale) [inline src/Calendar/Calendar.js:376]: 🔥

File: src/Calendar/Calendar.js [hunk 4]
```diff
@@ -519,6 +563,7 @@ class Calendar extends Component {
                         data-is-focused={day.isSame(currentDateDisplayed)}
                         key={copyDate}
                         onClick={isEnabledDate(day, this.props) ? () => this.dateClick(copyDate, enableRangeSelection) : null}
+                        onFocus={this.handleDayFocus(day)}
```
Comment (skvale) [inline src/Calendar/Calendar.js:566]: should this be `onFocus={() => this.handleDayFocus(day)}`?

File: src/Calendar/Calendar.js [hunk 5]
```diff
@@ -397,6 +419,26 @@ class Calendar extends Component {
         return blockedDates[0].isBefore(date, 'day') && blockedDates[1].isAfter(date, 'day');
     }
 
+    isWeekend(date) {
+        return date.day() % 6 === 0;
+    }
+
+    isFocusedDay(date) {
+        return this.state.currentFocusDay.isSame(date);
+    }
```
Comment (skvale) [inline src/Calendar/Calendar.js:428]: How about we keep a standard style for class functions
1
```js
    isFocusedDay(date) {
        return this.state.currentFocusDay.isSame(date);
    }
```
vs
2
```js
    isFocusedDay = date => {
        return this.state.currentFocusDay.isSame(date);
    }
```
I prefer `#2`

File: src/Calendar/Calendar.js [hunk 6]
```diff
@@ -397,6 +419,26 @@ class Calendar extends Component {
         return blockedDates[0].isBefore(date, 'day') && blockedDates[1].isAfter(date, 'day');
     }
 
+    isWeekend(date) {
+        return date.day() % 6 === 0;
```
Comment (skvale) [inline src/Calendar/Calendar.js:423]: Not a big deal, but I wonder if 
```js
return [0, 6].includes(date.day());
```
would be easier to understand

File: src/Calendar/Calendar.js (hunk 4 above)
Comment (meganaconley) [reply src/Calendar/Calendar.js:566]: Lint complains about creating a function in a loop. This is why the function declarations you pointed out seem weird as well. Yes, even though the `onClick` looks the same, just the focus one triggered the lint rule.

File: src/Calendar/Calendar.js [hunk 7]
```diff
@@ -659,12 +714,14 @@ Calendar.propDescriptions = {
...
     focusOnInit: 'Set to **true** to focus the calendar grid upon being mounted',
     localizedTextShape: {
+        calendarInstructions: 'Localized string informing screen reader users the calendar can be navigated by arrow keys.',
         nextMonth: 'aria-label for next button',
         previousMonth: 'aria-label for previous button',
         show12NextYears: 'aria-label for next button when years are displayed',
         show12PreviousYears: 'aria-label for previous button when years are displayed'
     },
     monthListProps: 'Additional props to be spread to the month\'s `<table>` element.',
+    specialDays: 'Object with special dates and special date types in shape of `{moment(date).format(\'YYYYMMDD\'): type}`. Type must be a number between 1-20.',
```
Comment (jacobdevera) [inline src/Calendar/Calendar.js:724]: Just glancing at this, it's a little hard to tell how I should be passing a special day in as a prop. I think an example would help (with different types to see some of the different colors from 1-20).

Comment (meganaconley) [reply src/Calendar/Calendar.js]: I updated the description here and added a few more examples to the component page. Does that seem better or should there be more examples?

Comment (jacobdevera) [reply src/Calendar/Calendar.js]: Yep, the description change and examples make it a lot more clear 👍

**Pull request merged**